ClosedXML/Excel/XLWorkbook_Load.cs

```diff
@@ -749,7 +749,9 @@ private void LoadSpreadsheetDocument(SpreadsheetDocument dSpreadsheet)
 
                                         if (df.BaseField?.Value != null)
                                         {
-                                            var col = pt.SourceRange.Column(df.BaseField.Value + 1);
+                                            IXLRangeColumn col = df.BaseField.Value == 0
```

Comment:
Can you please explain this?

Reply:
I saw this issue directly in this test. The source file has
		`<x:dataField name="Sum Amount paid" fld="6" subtotal="sum" showDataAs="normal" baseField="3" baseItem="0" numFmtId="0" />`
but saved file
		`<x:dataField name="Sum Amount paid" fld="6" subtotal="sum" showDataAs="normal" baseField="4" baseItem="0" numFmtId="0" />`

Reply:
So if `df.BaseField.Value == 0`, then you would be referencing `pt.SourceRange.Column(1)` (the first column), but if `df.BaseField.Value == 1`, then you would *also* be reference  `pt.SourceRange.Column(1)`?

Reply:
I had to go read up on what `BaseField` means. As per https://docs.microsoft.com/en-us/dotnet/api/documentformat.openxml.spreadsheet.datafield?view=openxml-2.8.1 it's used when the "`ShowDataAs` calculation is in use". So in your reference file, this attribute isn't relevant. But in the case where it is relevant, this change is wrong and should be reverted.

Reply:
But on the other hand, it is also not true that opening and saving a file changes this attribute. It should not be.